src/lava/magma/core/model/py/ports.py

```diff
@@ -344,7 +707,20 @@ def peek(self) -> ty.Tuple[int, int]:
 
 
 class RefVarTypeMapping:
-    """Class to get the mapping of PyRefPort types to PyVarPortTypes."""
+    """Class to get the mapping of PyRefPort types to PyVarPort types.
```

Comment:
Is the code in line 703 actually correct? Why do we need to specify these values at that point outside any class?

Reply:
see answer above